pkg/models/kubectl/kubectl.go

```diff
@@ -39,9 +40,18 @@ const (
 	namespace = constants.KubeSphereControlNamespace
 )
 
+var DefaultImage = "dockerhub.qingcloud.com/kubesphere/kubectl:advanced-1.0.0"
```

Comment:
move kubectl image to docker.io, cause dockerhub.qingcloud.com may not accessible everywhere